src/plugins/notebook/monkeyPatchObjectAPIForNotebooks.js

```diff
@@ -74,19 +74,22 @@ async function resolveNotebookTagConflicts(localAnnotation, openmct) {
 
 async function resolveNotebookEntryConflicts(localMutable, openmct) {
     if (localMutable.configuration.entries) {
+        const FORCE_REMOTE = true;
```

Comment:
See previous comment. Would it read better if we do something like `USE_CACHE = false` ?

Reply:
I definitely understand what you mean. This will hopefully just be a temporary flag.